### PR TITLE
[ios, macos] Update Jazzy theme

### PR DESCRIPTION
**Pull request by captainbarbosa**
Edits Jazzy config to support https://github.com/realm/jazzy/pull/1077.

Requires https://github.com/mapbox/jazzy-theme/pull/9 to merge before merging.

cc @1ec5

### CODE REVIEW

**Comment by friedbunny** [comment]
Couple additional steps that are necessary:

- Bump [the theme submodule](https://github.com/mapbox/mapbox-gl-native/tree/cdd47c035807583c0437bd75a0deb3347037042d/platform/darwin/docs) here after merging upstream.
- Bump [the jazzy version](https://github.com/mapbox/mapbox-gl-native/blob/cdd47c035807583c0437bd75a0deb3347037042d/platform/ios/scripts/install-packaging-dependencies.sh#L5).

**Comment by captainbarbosa** [comment]
Done 🔧

**Comment by friedbunny** [comment]
Jazzy’s still not liking this:

```
* Generating jazzy docs for 5.2.0-alpha.3…
/Library/Ruby/Gems/2.3.0/gems/jazzy-0.10.0/lib/jazzy/config.rb:432:in `parse_command_line': invalid option: --module_version (OptionParser::InvalidOption)
	from /Library/Ruby/Gems/2.3.0/gems/jazzy-0.10.0/lib/jazzy/config.rb:389:in `parse!'
	from /Library/Ruby/Gems/2.3.0/gems/jazzy-0.10.0/bin/jazzy:15:in `<top (required)>'
	from /usr/local/bin/jazzy:22:in `load'
	from /usr/local/bin/jazzy:22:in `<main>'
make[1]: *** [idocument] Error 1
make: *** [iframework] Error 2
Exited with code 2
```

File: platform/macos/scripts/document.sh [hunk 1]
```diff
@@ -47,7 +47,6 @@ jazzy \
     --readme ${README} \
     --documentation="platform/{darwin,macos}/docs/guides/*.md" \
     --theme platform/darwin/docs/theme \
-    --output ${OUTPUT}
-# https://github.com/realm/jazzy/issues/411
-find ${OUTPUT} -name *.html -exec \
-    perl -pi -e 's/BRANDLESS_DOCSET_TITLE/Maps SDK for macOS $1/, s/Mapbox\s+(Docs|Reference)/Mapbox Maps SDK for macOS $1/' {} \;
+    --output ${OUTPUT} \
+    --title "Maps SDK for macOS" \
+    --module_version ${SHORT_VERSION}
```
Comment (friedbunny) [inline platform/macos/scripts/document.sh]: Also fix the typo here.
```suggestion
    --module-version ${SHORT_VERSION}
```

**Pull request merged**